dev/Common/Microsoft.Utf8.h

```diff
@@ -45,8 +49,52 @@ template<> inline winrt::hstring ToUtf16(PCSTR utf8)
     return winrt::hstring(s.get());
 }
 
+inline std::wstring ToUtf16(const std::string& utf8)
+{
+    return ToUtf16(utf8.c_str());
+}
+
 inline winrt::hstring ToHString(PCSTR utf8)
 {
     return ToUtf16<winrt::hstring>(utf8);
 }
+
+// UTF16->UTF8 conversions
```

Comment:
@DrusTheAxe  could you clarify why are all these needed / why don't we use `winrt::to_hstring` and `winrt::to_string` ?